pkg/tsdb/prometheus/models/query.go

```diff
@@ -75,9 +78,14 @@ type Query struct {
 	RangeQuery    bool
 	ExemplarQuery bool
 	UtcOffsetSec  int64
+	Scope         Scope
```

Comment:
Scope means many other things too and it is not obvious to someone who is not familiar with the feature. I think having a little comment line here will be quite useful